script/build-release/revision/revision.go

```diff
@@ -299,3 +300,19 @@ func GetVersionFiles(rootDir string) []string {
 		// GetRuntimeVersionFile(rootDir),
 	}
 }
+
+func CopyCoreVersion(xgoVersionFile string, runtimeVersionFile string) error {
+	// copy xgo's core version to runtime version
+	code, err := ioutil.ReadFile(xgoVersionFile)
```

Comment:
You kept ioutil for compatibility purpose?

https://pkg.go.dev/io/ioutil#ReadFile

I don't think you should care about go before 1.16

But if you need it, then please add a comment here about why you are using this deprecated function

Reply:
Yes I kept using ioutil for compatibility reason. I expect go1.16 to be the smallest version xgo can support

Reply:
Then it recommend a comment

Reply:
But wait ioutil was deprecated with 1.16, so?

Reply:
So I'll lower the smallest version to go1.14 😼

Reply:
Then a comment aside the import and one where you call ioutil.ReadFile would be appreciated

Reply:
That's just a joke, the project's `runtime/go.mod` really says it is targeting `go 1.14`, so kept here.

Maybe after a year or two we may finally remove this thing.